packages/transport/test/modules/publicRelationshipTemplateReferences/PublicRelationshipTemplateReferencesController.test.ts

```diff
@@ -0,0 +1,74 @@
+import { IDatabaseConnection } from "@js-soft/docdb-access-abstractions";
+import { AccountController, ClientResult, PublicRelationshipTemplateReferenceClient, RequestError, Transport } from "../../../src";
+import { RestClientMocker } from "../../testHelpers/RestClientMocker";
+import { TestUtil } from "../../testHelpers/TestUtil";
+
+let connection: IDatabaseConnection;
+
+let transport: Transport;
+let account: AccountController;
+let clientMocker: RestClientMocker<PublicRelationshipTemplateReferenceClient>;
+
+beforeAll(async function () {
+    connection = await TestUtil.createDatabaseConnection();
+    transport = TestUtil.createTransport(connection);
+
+    await transport.init();
+
+    const accounts = await TestUtil.provideAccounts(transport, 3);
+
+    account = accounts[0];
+
+    const client = (account.publicRelationshipTemplateReferences as any).client as PublicRelationshipTemplateReferenceClient;
+    clientMocker = new RestClientMocker(client);
+});
+
+afterAll(async () => {
+    await account.close();
+
+    await connection.close();
+});
+
+afterEach(() => {
+    clientMocker.reset();
+});
```

Comment:
why isn't this a one-liner?

Reply:
yeah, we should stick to codestyle from anywhere else and use the shorthand for oneliners